module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -267,6 +278,40 @@ public function supportsConnectingLibraryCard($authMethod = null)
             && $this->getAuth($authMethod)->supportsConnectingLibraryCard();
     }
 
+    /**
+     * Is persistent login supported?
+     *
+     * @param User $user optional; check user auth method in database
+     *
+     * @return bool
+     */
+    public function supportsPersistentLogin($user = null)
+    {
+        if (!empty($this->config->Authentication->persistent_login)) {
+            if ($this->getAuth() instanceof ChoiceAuth) {
```

Comment:
This could be simplified with a ternary:

```
$method = $this->getAuth() instanceof ChoiceAuth
    ? $this->getAuth()->getSelectedAuthOption() : $this->getAuthMethod();
```